src/publish-search.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const fs = require('fs-extra');
+const request = require('request');
+const path = require('path');
+const endpoint = process.env.searchEndpoint;
+const token = process.env.token;
+const filePath = path.join(process.cwd(), 'src', 'stache', 'search', 'search.json');
+
+function getSearchData() {
+    try {
+        let file = fs.readJsonSync(filePath);
+        return JSON.stringify(file);
+    } catch (error) {
+        throw new Error(`[ERROR]: Unable to read search file at ${filePath}! ${error.message}`);
+    }
+}
+
+function publishSearch(argv, config) {
+    if (!config.appSettings.stache.search || !fs.existsSync(filePath)) {
```

Comment:
What happens if the consumer hasn't set `appSettings` at all? Possibly wrap this in a `try/catch`...